### PR TITLE
Creation of Create, Load and Execute Actions on Entities documentation.

### PR DESCRIPTION
**Pull request by pande3**
sidebars.js updated to add Entities sub menu and the new files.

### CODE REVIEW

**Comment by pande3** [comment]
Sorry for so many files in this PR! There is repetitive text, so do check for things like accidentally referencing xlsx in the CSV doc. Just to make sure I changed the text correctly for each file.

File: docs/actions/load-entities-sql.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+- Run a SQL query for any other reason than selecting records to create a contextual entity. For other SQL query scenarios, use the [Run SQL Query](/docs/actions/run-sql) action instead.
+
+## `Related Actions`
+
+| Action Name | Description |
+| -- | -- |
+| [Execute Actions on Entity List](/docs/actions/execute-actions-entity-list)   | Execute a list of actions on each item in the specified context entity. Use this to process the items loaded in the Entity created. |
+| [Extend Entity](/docs/actions/extend-entity)   | Extends an entity with new properties. |
+| `Serialize to CSV`   | Serializes a list of entities from the context into CSV (Comma Separated Values) format. |
```
Comment (Teruah2540) [inline docs/actions/load-entities-sql.md:31]: Is it safe to assume this one and the following item don't have documentation written yet and that is why they are formatted this way?

**Review by Teruah2540 — APPROVED** [review]
The only thing I noticed as on the 'load entities sql' and I noted it there.   I believe it's just done that way because we don't have documentation yet for two of the actions but wanted to double check.

File: docs/actions/load-entities-sql.md (hunk 1 above)
Comment (pande3) [reply docs/actions/load-entities-sql.md:31]: Yes, that's correct. Perhaps I will get those next! I will go ahead and merge this one.

**Pull request merged**